### PR TITLE
Add schnorr functions

### PR DESCRIPTION
**Pull request by Tibo-lg**
## Linked Issue

NA

## Overview

Add schnorr functions 

## How to use

NA

## Items reserved this time, or TODO

NA

## Check list

** Person who issued **
- [X] checked script 
- [ X build successed
- [ ] Linked PullRequest and Issue

** Reviewer **
- [ ] (if necessary) Record the review results of related tickets.

### CODE REVIEW

File: src/input_json_format/cfdapi_get_schnorr_pubkey_from_privkey copy.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "comment": "Get a Schnorr pubkey from a privkey.",
+  "request": {
+    ":class": "GetSchnorrPubkeyFromPrivkeyRequest",
+    ":class:comment": "Request to get a Schnorr pubkey from privkey.",
+    "privkey": "",
+    "privkey:require": "require",
+    "privkey:comment": "privkey (wif or hex)",
+    "isCompressed": true,
+    "isCompressed:require": "optional",
+    "isCompressed:comment": "compressed pubkey flag"
```
Comment (k-matsuzawa) [inline src/input_json_format/cfdapi_get_schnorr_pubkey_from_privkey copy.json:12]: unused `isCompressed`.
```suggestion
    "privkey:comment": "privkey (wif or hex)"
```

**Review by k-matsuzawa — COMMENTED** [review]
I reviewed it.
It seems that unnecessary parameters are described in only one place, so it would be helpful if you could delete them. 🙏

File: src/input_json_format/cfdapi_get_schnorr_pubkey_from_privkey copy.json (hunk 1 above)
Comment (Tibo-lg) [reply src/input_json_format/cfdapi_get_schnorr_pubkey_from_privkey copy.json]: Thanks fixed (also fixed the name of the file).

Comment (Tibo-lg) [reply src/input_json_format/cfdapi_get_schnorr_pubkey_from_privkey copy.json]: Thanks fixed (also fixed the file name)

**Comment by Tibo-lg** [comment]
Finally managed to fix the lint errors. Let me know if there is any other issues :)

**Review by k-matsuzawa — APPROVED** [review]
LGTM

**Pull request merged**